river/cluster/clustream.py

```diff
@@ -32,6 +32,11 @@ class CluStream(base.Clusterer):
     the use of Welford's algorithm [^2] to calculate the incremental variance, facilitated
     through `stats.Var()` available within `River`.
 
+    Since `River` does not support an actual "off-line" phase of the clustering algorithm (as data points
+    are assumed to arrive continuously, one at a time), we introduce a new parameter `time_gap`. After each
```

Comment:
```suggestion
    are assumed to arrive continuously, one at a time), a `time_gap` parameter is introduced. After each
```

Reply:
Mark as resolved since `time_gap` is now omitted.